crates/zizmor/src/config_schema.rs

```diff
@@ -0,0 +1,202 @@
+use std::collections::HashMap;
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::config::{UsesPolicy, WorkflowRule};
+use crate::models::uses::RepositoryUsesPattern;
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema)]
```

Comment:
Q: is there a reason we need the `Serialize` trait on these types? FWICT we don't actually use that functionality.

Reply:
Oh huh, I'm kind of surprised it causes a dead code warning even though schemars accesses these. I can look into that. But thanks for removing the `Serialize` impls!